CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 1.6.0 (2025-02-19)
+
+### Fixed
+
+* [STOR-1583](https://issues.infn.it/jira/browse/STOR-1583) - Fix how a request with no remote IP address is logged
```

Comment:
FYK: I opened another issue to investigate the reasons behind the missing IP
https://issues.infn.it/jira/browse/STOR-1623